### PR TITLE
[WIP] Add PX4 advanced tuning info.

### LINKED ISSUE #136: Add very cool PID tuning stuff
Add docs: https://github.com/mavlink/qgroundcontrol/pull/6295

### PR DESCRIPTION
**Pull request by hamishwillee**
Fixes #136 
(ie adds support tor PX4 advanced tuning https://github.com/mavlink/qgroundcontrol/pull/6295)

Note, I may split into separate ArduPilot and PX4 docs, which is why this is still WIP.

### CODE REVIEW

File: en/SetupView/Tuning.md [hunk 1]
```diff
@@ -2,22 +2,25 @@
...
 
 ### AutoTune
 
 AutoTune is used to automatically tune the rate parameters in order to provide the highest response without significant overshoot.
 
 Performing an AutoTune:
 
-* Select which axes you would like to tune. Tuning all axes at once can take a significant amount of time which may cause you to run out of battery. In order to prevent this you can select to tune only one axis at a time. 
-* Assign AutoTune to one of your transmitter switches. Ensure that switch is in low position before taking off.
+* Select which axes you would like to tune.
```
[the remark this answers is not in the document]
Comment (DonLakeFlyer) [reply en/SetupView/Tuning.md]: It hasn't changed. Why remove those two bullets?

File: en/SetupView/Tuning.md [hunk 2]
```diff
@@ -29,13 +32,37 @@ Performing an AutoTune:
...
\ No newline at end of file
+### Basic Tuning
+
+Adjust the specified flight characteristic by moving the slider(s) to the left or right.
+
+![PX4 Tuning - Basic](../../assets/setup/tuning/px4_copter_basic.jpg)
+
+### Advanced Tuning
+
+> **Note** Advanced tuning is currently only available on PX4 Multicopter frames. <!-- Still only multirotors? -->
```
[the remark this answers is not in the document]
Comment (DonLakeFlyer) [reply en/SetupView/Tuning.md]: Multi-Rotor only. It's still a large WIP. I'm liklely to pull if out this next Stable and then turn it back on for new dailies.

File: en/SetupView/Tuning.md [hunk 3]
```diff
@@ -2,22 +2,25 @@
 
 This page allows you to configure settings on your vehicle which control basic flight characteristics. The details of the page differ slightly if you are using PX4 Pro firmware or ArduPilot firmware.
 
-### Basic Tuning
+## ArduCopter Tuning
 
-A set of Basic Tuning sliders are suppored for both firmwares. They allow you to adjust the specified flight characteristic by moving the slider to the left or right.
+![ArduCopter Tuning Page](../../assets/setup/tuning/arducopter.png)
```
[the remark this answers is not in the document]
Comment (hamishwillee) [reply en/SetupView/Tuning.md]: Yes, but is it the same screen for all cases?

File: en/SetupView/Tuning.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (hamishwillee) [reply en/SetupView/Tuning.md]: I _moved_ the bullets. The first one is immediately below and is now a **Tip** (visually more obvious). The second is still there, below the tip.

[the remark this answers is not in the document]
Comment (hamishwillee) [reply en/SetupView/Tuning.md]: This section now moved, but you can see my point above in https://github.com/mavlink/qgc-user-guide/pull/175/commits/573e2d3dc1c3b35168154633e0c9a744cdb46bdf

**Comment by hamishwillee** [comment]
@DonLakeFlyer This is now split into ardupilot and px4 specific docs. I've added the relevant frames for each bit of functionality in the headings.

Ready to merge IMO - OK?

File: en/SetupView/Tuning.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (DonLakeFlyer) [reply en/SetupView/Tuning.md]: Ah, got it

**Comment by DonLakeFlyer** [comment]
LGTM

**Pull request merged**